components/EntryContent/index.js

```diff
@@ -0,0 +1,49 @@
+import styled from "styled-components";
+
+function EntryContent({ entry, handleAddEntry, isEditMode, databaseMutate }) {
```

Comment:
You're not using `isEditMode` and `databaseMutate`.
Either remove them if you don't need them or make use of them =)